Classes/FormEngine/Container/QbankSelectorButtonContainer.php

```diff
@@ -189,4 +190,37 @@ protected function getExtensionConfigurationManager(): ExtensionConfigurationMan
 
         return $extensionConfigurationManager;
     }
+
+    /**
+     * Generate inline style attribute if needed.
+     * Partly based on render function of
+     * TYPO3\CMS\Backend\Form\Container\InlineControlContainer.
+     *
+     * @return string
+     */
+    protected function inlineStyleAttribute(): string
+    {
+        $inlineStyle = [];
+        $parameterArray = $this->data['parameterArray'];
+
+        $config = $parameterArray['fieldConf']['config'];
+        $isReadOnly = isset($config['readOnly']) && $config['readOnly'];
+
+        $numberOfFullLocalizedChildren = 0;
+        foreach ($parameterArray['fieldConf']['children'] as $child) {
+            if (!$child['isInlineDefaultLanguageRecordInLocalizedParentContext']) {
+                $numberOfFullLocalizedChildren++;
+            }
+        }
+
+        if ($isReadOnly || $numberOfFullLocalizedChildren >= $config['maxitems']) {
+            $inlineStyle[] = 'display: none';
+        }
+
+        if (count($inlineStyle) > 0) {
+            return 'style="' . implode(';', $inlineStyle) . '"';
```

Comment:
I think `$inlineStyles` (plural) is better for an array. I don't know what `$numberOfFullLocalizedChildren` really counts, but I imagine it's either "filled" (i.e. populated) or "fully" (i.e. completely) localized children.

```suggestion
        $inlineStyles = [];
        $parameterArray = $this->data['parameterArray'];

        $config = $parameterArray['fieldConf']['config'];
        $isReadOnly = isset($config['readOnly']) && $config['readOnly'];

        $numberOfFullyLocalizedChildren = 0;
        foreach ($parameterArray['fieldConf']['children'] as $child) {
            if (!$child['isInlineDefaultLanguageRecordInLocalizedParentContext']) {
                $numberOfFullyLocalizedChildren++;
            }
        }

        if ($isReadOnly || $numberOfFullyLocalizedChildren >= $config['maxitems']) {
            $inlineStyles[] = 'display: none';
        }

        if (count($inlineStyles) > 0) {
            return 'style="' . implode(';', $inlineStyles) . '"';
```